client/src/pages/InsightsPage/InsightsPage.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./InsightsPage.module.css";
+import { DateSelector } from "../../components/DateSelector/DateSelector.js";
+
+const InsightsPage = () => {
+  return (
+    <div className={styles.bg}>
+      <DateSelector
+        showTime
```

Comment:
I would set this to `showTime={true}` instead of just having `showTime`

Reply:
Changed as requested.